src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1117,6 +1122,8 @@ namespace WaterToAirHeatPumpSimple {
         SimpleWatertoAirHP(HPNum).EnergyLatent = 0.0;
         SimpleWatertoAirHP(HPNum).EnergySource = 0.0;
         SimpleWatertoAirHP(HPNum).COP = 0.0;
+        DataHeatBalance::HeatReclaimSimple_WAHPCoil(HPNum).AvailCapacity = 0.0;
+        DataHeatBalance::HeatReclaimSimple_WAHPCoil(HPNum).DesuperheaterReclaimedHeat = 0.0;
```

Comment:
Setting `DesuperheaterReclaimedHeat = 0.0` here means it will always be zero later at line 2385 when it is subtracted from QSource. 

Using the stratified_gshp.idf test file, I made a second version with the desuperheater removed. The annual sum for RES GSHP CLG COIL:Cooling Coil Source Side Heat Transfer Energy [J](Hourly) is the same in both runs.  I expected this variable to be smaller in the run with desuperheater.                             
                                                                                                                                       
Possible solutions are a) only set this to zero when FirstHVACIteration is true, or b) set this to zero where it's used in `CalcDesuperheaterWaterHeater` or c)?

Reply:
Oh sorry, I did check the plant impact one or two months ago, and it shows a small difference on hvac (might be caused by something else?). I tested now it's giving the same results, and I fixed it as your solution b) it became reasonable. Thanks very much for pointing out!